<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Index</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>login_icon.png</normaloff>login_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">text-decoration:underline;</string>
        </property>
        <property name="text">
         <string>Enter your choice:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton1">
        <property name="text">
         <string>Top Mailers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton2">
        <property name="text">
         <string>Top Dates on which you Recieved emails</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton3">
        <property name="text">
         <string>Top Days of Week on which Email Recieved</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton4">
        <property name="text">
         <string>Top Months in which you recieved max emails</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton5">
        <property name="text">
         <string>TopTime-Ranges in whic you recieve mails</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton6">
        <property name="text">
         <string>Time-Range when Top Milers mail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton7">
        <property name="text">
         <string>Custom Time Range for Top Mailers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton8">
        <property name="text">
         <string>Top Years in which you recieved mails</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton9">
        <property name="text">
         <string>Date on which maximum emails arrived till now</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
